### DISCUSSION TITLE
git add styles.css - Error

### DISCUSSION THREAD

**Discussion by Elmanhands** [hack4impact-calpoly/starter-pack#10] [Q&A] [answered]
On starter Pack Lab 29 I downloaded the full style sheet, but when trying to run "git add styles.css" I got the following error:

fatal: LF would be replaced by CRLF in styles.css

The solutions I am finding online don't seem to work for me. Any suggestions?

Thanks!

**Comment by khoa-l** [comment, marked as answer]
Would you try `git config --global core.autocrlf false` in your VSCode terminal? I think VSCode naturally sets line endings and it conflicts with the global git settings!

**Reply by Elmanhands** [reply]
Thank you that worked
